geometry_msgs/Pose2D pose
string frame_id
int32 pixel_radius
string target_frame_id
time stamp
---
geometry_msgs/Point point

